pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go

```diff
@@ -644,9 +644,12 @@ func TestTenantStreamingMultipleNodes(t *testing.T) {
 		},
 	}
 
+	args.SrcClusterSettings[`server.sqlliveness.ttl`] = `'5m'`
```

Comment:
could you explain why this helps?

Reply:
no :) I saw just once the connection being killed so I increased the ttl, but this was when I tried running this test under race, and we're still skipping under race so it's not really needed. BTW even if we want it this needs to be within `CreateTenantStreamingClusters`.

Reply:
the default is 40 seconds, this change (that was reverted) was increasing that ttl. Maybe I'm missing something?
and BTW, if we need to bump this we should only do this under race as you said in the other comment..